packages/editor/src/plugins/image/components/pixabay-image-search.tsx

```diff
@@ -0,0 +1,191 @@
+        `https://pixabay.com/api/?key=${apiKey}&q=${encodeURIComponent(searchQuery)}&image_type=photo`
+      )
+      const data = (await response.json()) as PixabayResponse
+      setImages(data.hits)
+    } catch (error) {
+      console.error('Error fetching images from Pixabay:', error)
+    } finally {
+      setIsSearching(false)
+    }
+  }
+
+  const debouncedSearch = useCallback(
+    debounce((searchQuery: string) => {
+      void handleSearch(searchQuery)
+    }, 300),
+    []
+  )
+
+  useEffect(() => {
+    if (query.length > 3) {
+      void debouncedSearch(query)
+    }
+    // Cancel the debounce if the component is unmounted or the query changes
+    return () => {
+      debouncedSearch.cancel()
+    }
+  }, [query, debouncedSearch])
+
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setQuery(e.target.value)
+  }
+
+  const handleTagClick = async (tag: string) => {
+    setQuery(tag)
+    await handleSearch(tag)
+  }
+
+  const handleKeyDown = async (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key !== 'Enter') return
+    await handleSearch(query)
+  }
+
+  const showTags = images.length === 0 && !isSearching
+
+  return (
+    <div className="max-h-[60vw] min-h-[20vw] pt-2">
+      <h2 className="mb-6 ml-10 mt-10 font-bold">{imageStrings.licenceFree}</h2>
+      {/* Search input */}
+      <div className="relative ml-10 w-[90%]">
+        <input
+          ref={inputRef}
+          type="text"
+          value={query}
+          onChange={handleInputChange}
+          placeholder={imageStrings.search}
+          onKeyDown={handleKeyDown}
+          className="w-full rounded-lg border-0 bg-yellow-100 py-2 pl-4 pr-10 text-gray-600"
+        />
+        {query && (
+          <div
+            className="absolute bottom-0 right-0 top-0 flex cursor-pointer items-center justify-center px-4 text-gray-400"
+            onClick={() => {
+              setQuery('')
+              setImages([])
+              setTimeout(() => {
+                inputRef.current?.focus()
+              })
+            }}
+          >
+            <FaIcon icon={faXmarkCircle} />
+          </div>
+        )}
+      </div>
+
+      {/* Search tags */}
+      {showTags && (
+        <div>
+          <div className="mb-6 mt-10 flex flex-wrap justify-center">
+            {Object.values(imageStrings.searchTags).map((tagKey) => (
+              <button
+                key={tagKey}
+                onClick={() => handleTagClick(tagKey)}
+                className="m-2 inline-block rounded-md bg-white px-2 py-1 text-sm text-black shadow-md"
+              >
+                {tagKey}
+              </button>
+            ))}
+          </div>
+        </div>
+      )}
+
+      <div
+        className={cn(
+          'max-h-[500px]',
+          'mt-4 flex flex-wrap px-8',
+          isLoadingImage && 'max-h-100 border-1 border border-red-500'
+        )}
+        style={{ overflow: isLoadingImage ? 'hidden' : 'auto' }}
```

Comment:
why is this using style and is not part of tailwind classes above?

Reply:
Fixed, thanks